demos/ota/ota_demo_core_http/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+set( DEMO_NAME "ota_demo_core_http" )
+
+# Include required library's source and header path variables.
+include( ${CMAKE_SOURCE_DIR}/libraries/aws/ota-for-aws-iot-embedded-sdk/otaFilePaths.cmake )
+include( ${CMAKE_SOURCE_DIR}/libraries/standard/coreMQTT/mqttFilePaths.cmake )
+include( ${CMAKE_SOURCE_DIR}/libraries/standard/coreHTTP/httpFilePaths.cmake )
+# Include backoffAlgorithm library file path configuration.
+include( ${CMAKE_SOURCE_DIR}/libraries/standard/backoffAlgorithm/backoffAlgorithmFilePaths.cmake )
+
+# Demo target.
+add_executable(${DEMO_NAME})
+
+# librt is required to use OTA OS POSIX port.
+find_library(LIB_RT rt REQUIRED)
+
+target_sources(
+    ${DEMO_NAME}
+    PRIVATE
+        "ota_demo_core_http.c"
+        "${DEMOS_DIR}/ota/common/src/mqtt_subscription_manager.c"
+        "${DEMOS_DIR}/ota/common/src/http_demo_utils.c"
+        ${OTA_SOURCES}
+        ${OTA_OS_POSIX_SOURCES}
+        ${OTA_MQTT_SOURCES}
+        ${OTA_HTTP_SOURCES}
+        ${MQTT_SOURCES}
+        ${MQTT_SERIALIZER_SOURCES}
+        ${HTTP_SOURCES}
+        ${BACKOFF_ALGORITHM_SOURCES}
+)
```

Comment:
```suggestion
)

```

Reply:
Changed cmake in  https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1425/commits/9f18e9905580d6e4067eaa4a215da9dbe1e2453a